app/src/ui/lib/text-box.tsx

```diff
@@ -115,6 +152,14 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
     }
   }
 
+  private onRef = (instance: HTMLInputElement) => {
```

Comment:
Fair point, will also update the `onInputRef` prop which has the same issue while I'm in here:

```tsx
  readonly onInputRef?: (instance: HTMLInputElement) => void
```